packages/shorebird_cli/lib/src/commands/commands.dart

```diff
@@ -2,6 +2,7 @@ export 'account/account.dart';
 export 'apps/apps.dart';
 export 'build/build_command.dart';
 export 'cache/cache.dart';
+export 'channel_command.dart';
```

Comment:
We'll also need to add the new command to the `CommandRunner` in order for it to be usable.